app/src/test/java/java_server/RoutesTest.java

```diff
@@ -83,4 +93,28 @@ public void returnsANullPointerExceptionStringIfURIDoesntExistInDirectory() thro
 
         Assert.assertEquals(false, routes.isAnExistingFileInDirectory(fileList, request));
     }
+
```

Comment:
No es de este commit pero en la linea 66

**throwsNullPointerException**_WhenDirectoryDoesNotExist <- Ojo, NullPointerEx nunca es claro su origen, evitala

> Don’t just return an exception
> When returning an exception you should also provide a context of the error, why it happened, where it happened, source, location, etc. Mention the operation that failed and the type of failure.

https://victorelizalde.medium.com/objects-and-data-structures-error-handling-and-boundaries-in-clean-code-313472a6bec0

Reply:
Listo